### ISSUE TITLE
raisely local redirecting to live site

### ISSUE THREAD

**Issue by jasloe** [raisely/cli#54]
I am unable to get `raisely local` to work. There are three other reports of an issue with `raisely local` redirecting to the live site [in this issue](https://github.com/raisely/cli/issues/52). 

Creating a new issue with the hopes of getting fresh eyes on this. Attempted the very well documented fix [here](https://github.com/raisely/cli/issues/52#issuecomment-2301477553) but, unfortunately, not getting expected result.

In summary:

- Local dev server (http://localhost:8015) redirects to public my-campaign.raiselysite.com
- Not using a custom domain
- Changes made in local scss files (i.e. `my-campaign.scss`) are not detected by the CLI

Documentation reads:

```
With your project initialised, you're ready to start making changes. Run raisely local and select the campaign you're working on.

The CLI will boot up a local server, and open your campaign in your browser. The URL will be http://localhost:8015/.

Now, you can edit any stylesheets or components. Your campaign will automatically reload and reflect these changes, however they will not be visible on the live Raisely website.
```

Am I missing something fundamental?

FWIW, running node v20.18.0 (via nvm) and npm 10.9.0 on Monterey.

**Comment by jasloe** [comment]
This is fixed. Please create a release. Until that happens, for other's seeking a solution clone the repo: 

`~/.nvm/versions/node/[your-node-version]/bin/`

then

`yarn install`

then 

`raisely init` and `raisely local`, per the docs.

Make sure you create a `scss` file in the stylesheets directory with the name of you campaign, i.e. `.stylesheets/my-campaign.scss`.

**Comment by pku2319** [comment]
Released 1.8.3 to address this issue
